### ISSUE TITLE
Breadboard will not load any additional images...

### ISSUE THREAD

**Issue by ebonydad** [cocktailpeanut/breadboard#41]

**Comment by medledan** [comment]
Yeah, for me it takes forever to load up anything. Constantly refreshes itself (why), so it has to reindex; half the time it locks up, and when I try to close it, it freezes and I need to close it via Task Manager. It is basically completely broken.

**Comment by cocktailpeanut** [comment]
@ebonydad @medledan sorry about the problem.

Can you guys try the latest nightly release and let me know if it fixes the problem? https://github.com/cocktailpeanut/breadboard-staging/releases/tag/0.2.4

by the way "constantly refreshes itself" sounds like a completely different problem, hopefully the new version solves this but if it doesn't, please create an issue with details.

**Comment by cocktailpeanut** [comment]
Also, if you want a more realtime feedback and quicker fix, feel free to join the discord and share your situation. That way we can go back and forth much more quickly and probably be able to figure out the issue faster than going back and forth on this thread.

**Comment by medledan** [comment]
> Also, if you want a more realtime feedback and quicker fix, feel free to join the discord and share your situation. That way we can go back and forth much more quickly and probably be able to figure out the issue faster than going back and forth on this thread.

Can you refresh the discord link, getting invalid? maybe add it to this tread too?

**Comment by cocktailpeanut** [comment]
@medledan The link is https://discord.gg/6MJ6MQScnX - is this what you have? Anyway see you there!

**Comment by medledan** [comment]
> https://discord.gg/6MJ6MQScnX

Yup, keeps saying invalid 🤷‍♂️

**Comment by cocktailpeanut** [comment]
@medledan can you try this link? I dont know why but somehow it did expire so I created a new one https://discord.gg/XahBUrbVwz

**Comment by medledan** [comment]
> @medledan can you try this link? I dont know why but somehow it did expire so I created a new one https://discord.gg/XahBUrbVwz

Worked!

**Comment by ebonydad** [comment]
> @ebonydad @medledan sorry about the problem.
> 
> Can you guys try the latest nightly release and let me know if it fixes the problem? https://github.com/cocktailpeanut/breadboard-staging/releases/tag/0.2.4

Will try the update today. I fixed the problem initially by removing the folder in question, then reindexing it. Now I have gotten the the problem of running out of memory at times.

**Comment by cocktailpeanut** [comment]
This should be fixed with 0.4.0 right? https://www.reddit.com/r/StableDiffusion/comments/111dc2l/breadboard_040_realtime_messenger_for_stable/

Please feel free to close this was fixed. If not, let me know.
